torch/distributed/fsdp/flat_param.py

```diff
@@ -520,6 +520,10 @@ def shard_metadata(
             self.flat_param._shard_param_offsets[:],  # type: ignore[attr-defined]
         )
 
+    def _flat_param_to(self, *args, **kwargs):
```

Comment:
Right now, this method is not doing much, but this will be a critical helper method when exposing original parameters. I think it is fine to start migrating early.